nemo_skills/inference/generate.py

```diff
@@ -213,10 +218,11 @@ def generate(cfg: GenerateSolutionsConfig):
                 if cfg.multi_turn_key is None:
                     outputs = llm.generate(
                         prompts=[prompt.fill(dp) for dp in data_points],
-                        stop_phrases=prompt.stop_phrases,
+                        stop_phrases = (prompt.stop_phrases if prompt.stop_phrases is None else (prompt.stop_phrases + extra_stop_phrases)),
```

Comment:
why such condition? If it's None we should still take the extra stop phrases, right?

Reply:
It sims like mt-bench test in test_generations expect None as an input and are failing, when trying to pass empty array